app/docs/0.13.x/proxy.md

```diff
@@ -845,51 +886,53 @@ the client:
+streaming nature of Nginx. Each chunk of the upstream response that is
+successfully processed by such `body_filter` hooks is sent back to the client.
+You can find more informations about the `body_filter` hook in the [Plugin
+development guide][plugin-development-guide].
 
 [Back to TOC](#table-of-contents)
 
-## Configuring a fallback API
+## Configuring a fallback Route
 
 As a practical use-case and example of the flexibility offered by Kong's
-proxying capabilities, let's try to implement a "fallback API", so that in
-order to avoid Kong responding with an HTTP `404`, "API not found", we can
-catch such requests and proxy them to a special upstream service of yours, or
-apply a plugin to it (such a plugin could, for example, terminate the request
-with a different status code or response without proxying the request).
+proxying capabilities, let's try to implement a "fallback Route", so that in
+order to avoid Kong responding with an HTTP `404`, "no route found", we can
+catch such requests and proxy them to a special upstream service, or apply a
+plugin to it (such a plugin could, for example, terminate the request with a
+different status code or response without proxying the request).
 
-Here is an example of such a fallback API:
+Here is an example of such a fallback Route:
 
 ```json
 {
-    "name": "root-fallback",
-    "upstream_url": "http://dummy.com",
-    "uris": ["/"]
+    "paths": ["/"],
+    "service": {
+        "id": "..."
+    }
 }
 ```
 
-As you can guess, any HTTP request made to Kong would actually match this API,
-since all URIs are prefixed by the root character `/`. As we know from the
-[Request URI][proxy-request-uri] section, the longest URIs are evaluated first
-by Kong, so the `/` URI will eventually be evaluated last by Kong, and
-effectively provide a "fallback" API, only matched as a last resort.
+As you can guess, any HTTP request made to Kong would actually match this
+Route, since all URIs are prefixed by the root character `/`. As we know from
+the [Request path][proxy-request-path] section, the longest URL paths are
+evaluated first by Kong, so the `/` path will eventually be evaluated last by
+Kong, and effectively provide a "fallback" Route, only matched as a last
+resort. You can then send traffic to a special Service or apply any plugin you
+wish on this Route.
 
 [Back to TOC](#table-of-contents)
 
-## Configuring SSL for an API
+## Configuring SSL for a Route
 
 Kong provides a way to dynamically serve SSL certificates on a per-connection
-basis. Starting with 0.10, the SSL plugin has been removed and SSL certificates
-are directly handled by the core, and configurable via the Admin API. Your
-client HTTP library must support the [Server Name Indication][SNI] extension to
-make use of this feature.
+basis. SSL certificates are directly handled by the core, and configurable via
+the Admin API. Your client HTTP library must support the [Server Name
```

Comment:
suggestion: Your client HTTP library -> Clients connecting to Kong via HTTPS

(just to make evident that the library in question is not part of Kong)